### PR TITLE
Tung Nguyen - finished all java day 2 exercises + screenshots of demo and added pro…

### PR DESCRIPTION
**Pull request by tunggnguyen**
1. Finished all exercises from java day 2 with screenshots.
2. Added problem descriptions for exercises from yesterday

### CODE REVIEW

**Review by myolap — APPROVED** [review]

File: tungnguyen/java/BookTest.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+public class BookTest {
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        System.out.print("Enter book tittle: ");
+        String name = scan.nextLine();
+        System.out.print("Enter book id: ");
+        int id = Integer.valueOf(scan.nextLine());
+        Random random = new Random();
+        Book tempBook = new Book(id, name, random.nextInt(100));
+        SpecialEditionBook tempBook2 = new SpecialEditionBook(id, name, random.nextInt(1000), 100);
```
Comment (madhanramu) [inline tungnguyen/java/BookTest.java:15]: you need to get the discount from the user.

File: tungnguyen/java/Rectangle.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ * breadth.
+ * 
+ * Create a method area() which finds the area of the rectangle.
+ * 
+ * Note: Use constructors to assign values to datamembers
+ * 
+ * 
+ */
+public class Rectangle {
+    private int lengthOf;
```
Comment (madhanramu) [inline tungnguyen/java/Rectangle.java:15]: use double

File: tungnguyen/java/Rectangle.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    private int widthOf;
+
+    public Rectangle(int x, int y) {
+        lengthOf = y;
+        widthOf = x;
+    }
+
+    public double calculateArea() {
+        return lengthOf * widthOf;
```
Comment (madhanramu) [inline tungnguyen/java/Rectangle.java:24]: how the multiplication of two ints result in double. use double for length and width

File: tungnguyen/java/Student.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+    public void setStudDetails(String studName, int rollNum, int mark1, int mark2, int mark3) {
+        this.rollNum = rollNum;
+        this.studName = studName;
+        this.mark1 = mark1;
+        this.mark2 = mark2;
+        this.mark3 = mark3;
+    }
+
+    public int calculateTotal() {
```
Comment (madhanramu) [inline tungnguyen/java/Student.java:33]: make it private

File: tungnguyen/java/Day1Exercises.java [hunk 5]
```diff
@@ -12,50 +15,72 @@ public class Day1Exercises {
      * @param args
      */
     public static void main(String[] args) {
```
Comment (madhanramu) [inline tungnguyen/java/Day1Exercises.java:17]: Good work on this

**Review by madhanramu — CHANGES_REQUESTED** [review]

**Review by madhanramu — APPROVED** [review]

**Pull request merged**